opm/common/data/SimulationDataContainer.hpp

```diff
@@ -86,6 +94,7 @@ namespace Opm {
         const std::vector<double>& facepressure() const;
         const std::vector<double>& faceflux    () const;
 
+        const std::unordered_map<std::string, std::vector<double>> cellData() const;
```

Comment:
Did you really mean to return a copy of the entire value map here?  Should there not be an ampersand at the end of the return type?